src/js/view/utils/request-fullscreen-helper.js

```diff
@@ -30,7 +30,9 @@ export default function(elementContext, documentContext, changeCallback) {
             return supportsDomFullscreen;
         },
         requestFullscreen: function() {
-            requestFullscreen.apply(elementContext);
+            requestFullscreen.call(elementContext, {
+                navigationUI: 'hide'
```

Comment:
Are we sure this is ignored by browsers that do not support fullscreen options?
https://developer.mozilla.org/en-US/docs/Web/API/Element/requestFullScreen

Reply:
I would imagine it would treat it as if it didn't exist - not sure what browser would have an undocumented option for the func that they don't expect people to pass in. It's a guess all things considered but I'm fairly confident in that and figured any issue would come up in testing (automated tests looked good at least)

Reply:
test all the browsers